src/registrar/templates/includes/domains_table.html

```diff
@@ -135,6 +172,19 @@ <h2>Status</h2>
               >Deleted</label
             >
           </div>
+          {% if has_domain_renewal_flag and has_expiring_domains > 0 %}
```

Comment:
has_expiring_domains implies we're looking for a boolean value so I suggest refactoring this to num_expiring_domains or if we want to keep has_expiring_domains, making another getter to indicate this value >0

Reply:
updated